Dockerfile

```diff
@@ -1,26 +1,31 @@
 ARG GOLANG_VERSION=1.19
-ARG MENDER_CLI_VERSION=master
-ARG MENDER_ARTIFACT_VERSION=master
-ARG MENDER_CLIENT_VERSION=master
+ARG MENDER_CLI_VERSION=1.9.0
+ARG MENDER_ARTIFACT_VERSION=3.9.0
+ARG MENDER_CLIENT_VERSION=3.4.0
 
 FROM golang:${GOLANG_VERSION} as cli-builder
 WORKDIR /go/src/github.com/mendersoftware/mender-cli
 ARG MENDER_CLI_VERSION
-RUN git clone -b $MENDER_CLI_VERSION https://github.com/mendersoftware/mender-cli.git . && \
+RUN git clone https://github.com/mendersoftware/mender-cli.git . && \
+    git checkout $MENDER_CLI_VERSION && \
```

Comment:
It looks like lies in surface of personal preference, but just out of curiosity why do you prefer to do it this way?

Reply:
It is not personal preference, it just doesn't work for tags. You cannot checkout a tag with `git clone --branch` option

Reply:
I also wrote it into the commit message. Should I rephrase?